ingestors/arena-ingestor/src/main/kotlin/no/nav/amt/tiltak/ingestors/arena/domain/ArenaData.kt

```diff
@@ -22,7 +22,11 @@ internal data class ArenaData(
 		ingestedTimestamp = LocalDateTime.now()
 	)
 
-	fun markAsFailed() = this.copy(ingestStatus = IngestStatus.FAILED)
+	fun markAsFailed() = this.copy(
+		ingestStatus = IngestStatus.FAILED,
+		ingestAttempts = ingestAttempts + 1,
+		lastRetry = LocalDateTime.now()
+	)
```

Comment:
Kan være greit å gjenspeile endringen i `markAsFailed() should mark data as failed`-testen også slik at testen sjekker inkrement og dato

Reply:
fikset